### DISCUSSION TITLE
byte array []u8 needs to  concatenate with a +

### DISCUSSION THREAD

**Discussion by David-Worboys** [vlang/v#15163] [Ideas]
I think byte arrays []u8 need to concatenate with the + operator just like strings

**Comment by JalonSolov** [comment]
No, because it would be ambiguous.

`[1, 2] + [3, 4]` could mean either `[1, 2, 3, 4]` or it could mean `[4, 6]`.

Just use the existing `<<` operator.  Then it is very clear that `[1, 2] << [3, 4]` means `[1, 2, 3, 4]`.

**Comment by David-Worboys** [comment]
I can agree with that.  

I was thinking that << is not obvious and a + or join method would make it cleaner and easier to mentally grok when reading the code - particularly by newbies.

**Comment by David-Worboys** [comment]
Thinking it over, my issue arises because I am used to << as bit shift so when I read it I think bit shift not join or append. Would others from C++ backgrounds have the same issue?
